travis.sh

```diff
@@ -31,3 +32,7 @@ if [ -f $BOOST ]; then
     cucumber examples/Calc
     wait
 fi
+if [ -f $CGREEN ]; then
+    $CGREEN >/dev/null &
+    cucumber examples/FizzBuzz
```

Comment:
Please add a `wait` call after `cucumber`. That makes sure that to wait for `$CGREEN` to terminate (and to use its exit code).